src/main/java/edu/kis/powp/jobs2d/command/ICanvasBoundariesCheckStrategy.java

```diff
@@ -0,0 +1,5 @@
+package edu.kis.powp.jobs2d.command;
+
+public interface ICanvasBoundariesCheckStrategy {
+	boolean checkExceedingBoundaries(ICanvas canvas, int X, int Y);
```

Comment:
IMO: canvas parameter is not necessary here. ICanvas object can be simply a attribute of ICanvasBoundariesCheckStrategy implementation.